movie_REST_API/movie_REST_API/Controllers/InfoMovieViewController.swift

```diff
@@ -0,0 +1,278 @@
+// SecondViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Страница выбранного фильма
+final class InfoMovieViewController: UIViewController {
+    
+    // MARK: - Private Constant
+    private enum Constant {
+        static let cellIdentifier = "cell"
+        static let errorDataTaskString = "DataTask error: "
+        static let emptyDataString = "Empty Data"
+        static let firstPartURLString =  "https://image.tmdb.org/t/p/w500"
+        static let starImageName = "star"
+        static let starFillImageName = "star.fill"
+        static let emptyString = ""
+        static let addFavouriteString = "Фильм добавлен в избранное"
+        static let deleteFavouriteString = "Фильм удален из избранного"
+        static let baseImageName = "фон5"
+        static let watchString = "Смотреть"
+    }
+    
+    // MARK: - Private Visual Components
+    private let imageCollectionView: UICollectionView = {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .horizontal
+        let collection = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collection.backgroundColor = .none
+        collection.showsHorizontalScrollIndicator = false
+        collection.translatesAutoresizingMaskIntoConstraints = false
+
+        return collection
+    }()
+    
+    private let movieImageView: UIImageView = {
+        let image = UIImageView()
+        image.layer.cornerRadius = 15
+        image.clipsToBounds = true
+        image.translatesAutoresizingMaskIntoConstraints = false
+        return image
+    }()
+    
+    private let goToWebButton: UIButton = {
+        let button = UIButton()
+        button.setTitle( Constant.watchString, for: .normal)
+        button.layer.cornerRadius = 10
+        button.backgroundColor = .systemPurple
+        button.setTitleColor(UIColor.black, for: .normal)
+        button.clipsToBounds = true
+        button.layer.borderWidth = 1
+        button.layer.borderColor = UIColor.purple.cgColor
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    let descpriptionTextView: UITextView = {
+        let text = UITextView()
+        text.font = .systemFont(ofSize: 22)
+        text.backgroundColor = .none
+        text.textColor = .black
+        text.textAlignment = .center
+        text.showsVerticalScrollIndicator = false
+        text.translatesAutoresizingMaskIntoConstraints = false
+        return text
+    }()
+    
+    let nameFilmLabel: UILabel = {
+        let label = UILabel()
+        label.font = .boldSystemFont(ofSize: 25)
+        label.textColor = .black
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    // MARK: - Private Properties
+    private let actorViewModel = ActorViewModel()
```

Comment:
```suggestion
    private let actorView = ActorViewModel()
```
мы не используем слово Model!